VineyardInternal/Resolution.swift

```diff
@@ -11,6 +11,17 @@ enum TimeBound: String, CaseIterable {
     case day
```

Comment:
nit: You could instead use the rawValue since the TimeBound enum is a String anyway.

```swift
enum TimeBound: String, CaseIterable {
  case day = "Daily"
  // ...
}

// in the view (or wherever description is used):

currTimeBound.rawValue // instead of currTimeBound.description()
```